sqlmesh/integrations/github/cicd/controller.py

```diff
@@ -0,0 +1,680 @@
+                    model_rows = [
+                        h("td", affected_model.model_name),
+                        h("td", affected_model.change_category_str),
+                    ]
+                    if affected_model.intervals:
+                        model_rows.append(h("td", affected_model.formatted_loaded_intervals))
+                    else:
+                        model_rows.append(h("td", "N/A"))
+                    body_rows.append(model_rows)
+                table_header = h("thead", [h("tr", row) for row in header_rows])
+                table_body = h("tbody", [h("tr", row) for row in body_rows])
+                summary = str(h("table", [table_header, table_body]))
+            self._update_check(
+                name="SQLMesh - PR Environment Synced",
+                status=status,
+                conclusion=conclusion,
+                title=title,
+                summary=summary,
+            )
+            self.update_sqlmesh_comment_info(
+                value=f"- PR Virtual Data Environment: `{self.pr_environment_name}`",
+                find_regex=r"- PR Virtual Data Environment: `.*`",
+                replace_if_exists=False,
+            )
+        else:
+            conclusion_to_summary = {
+                GithubCheckConclusion.SKIPPED: f":next_track_button: Skipped creating or updating PR Environment `{self.pr_environment_name}` since a prior stage failed",
+                GithubCheckConclusion.FAILURE: f":x: Failed to create or update PR Environment `{self.pr_environment_name}`. There are likely uncateogrized changes. Run `plan` to apply these changes.",
+                GithubCheckConclusion.CANCELLED: f":stop_sign: Cancelled creating or updating PR Environment `{self.pr_environment_name}`",
+                GithubCheckConclusion.ACTION_REQUIRED: f":warning: Action Required to create or update PR Environment `{self.pr_environment_name}`. There are likely uncateogrized changes. Run `plan` to apply these changes.",
+            }
+            summary = conclusion_to_summary.get(
+                conclusion, f":interrobang: Got an unexpected conclusion: {conclusion.value}"
+            )
+            self._update_check(
+                name="SQLMesh - PR Environment Synced",
+                status=status,
+                conclusion=conclusion,
+                title=title,
+                summary=summary,
+            )
+
+    def update_prod_environment_check(
+        self, status: GithubCheckStatus, conclusion: t.Optional[GithubCheckConclusion] = None
+    ) -> None:
+        """
+        Updates the status of the merge commit for the prod environment.
+        """
+        status_to_title = {
+            GithubCheckStatus.IN_PROGRESS: "Deploying to Prod",
+            GithubCheckStatus.QUEUED: "Waiting to see if we can deploy to prod",
+        }
+        title = summary = status_to_title.get(status)
+        if not title:
+            assert conclusion
+            conclusion_to_title = {
```

Comment:
move these to constants

Reply:
See this: https://github.com/TobikoData/sqlmesh/pull/810#discussion_r1185329261